src/Microsoft.DotNet.Interactive.Formatting/HtmlFormatter{T}.cs

```diff
@@ -186,7 +192,17 @@ bool BuildTable(T source, FormatContext context, bool summarize)
                     }
                 }
 
-                rows.Add(tr(rowValues.Select(r => td(r))));
+                rows.Add(tr(rowValues.Select(value =>
```

Comment:
do we need any change on the TabularDataResource Formatter?

Reply:
What kind of change?